beta-src/src/components/WDInfoDisplay.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from "react";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  TableContainer,
+} from "@mui/material";
+// import { makeStyles } from "@mui/core/styles";
```

Comment:
make styles is deprecated. please remove. you can use sx or [styled](https://mui.com/system/styled/)

Reply:
> make styles is deprecated. please remove. you can use sx or [styled](https://mui.com/system/styled/)

I removed this optional styling from comments. and I used sx for all mui components. Thanks!